irohad/main/server_runner.cpp

```diff
@@ -5,15 +5,22 @@
 
 #include "main/server_runner.hpp"
 
+#include <fstream>
 #include <boost/format.hpp>
 #include "logger/logger.hpp"
 
 const auto kPortBindError = "Cannot bind server to address %s";
+const auto kPrivateKeyError = "Cannot read private key file";
+const auto kCertificateError = "Cannot read certificate file";
 
 ServerRunner::ServerRunner(const std::string &address,
                            logger::LoggerPtr log,
-                           bool reuse)
-    : log_(std::move(log)), serverAddress_(address), reuse_(reuse) {}
+                           bool reuse,
+                           const std::string &tls_keypair)
+    : log_(std::move(log)),
+      serverAddress_(address),
+      reuse_(reuse),
+      tlsKeypair_(tls_keypair) {}
```

Comment:
tls_keypair_. serverAddress_ can be refactored too since you have been touched that part of the code :)

Reply:
Fixed in latest version